provision/dev/nginx_site_docker.conf

```diff
@@ -4,7 +4,7 @@ server {
 
   location / {
   proxy_set_header X-Forwarded-Email "aaron.snow@gsa.gov";
-    proxy_set_header Host 192.168.33.10;
+    proxy_set_header Host 192.168.99.100:8000;
```

Comment:
@toolness:  We (@batemapf and I) were running into an issue when viewing the project details pages and when saving changes to a project in the admin.  This `nginx` config line is the culprit, but do you have any idea why this is needed?  Alternatively, what had you pulled from to set these values here, the `Vagrant` config?

Also, is it possible to somehow set this dynamically?  What I'm wondering is if we could somehow set it to whatever the value is of `docker-machine ip default` when run on anyone's local machine, in case the IP address is different, and use the `EXTERNAL_PORT` environment variable defined in the `.env` file.

The best answer I could find to this was in the [nginx Docker image documentation](https://hub.docker.com/_/nginx/) (under the "using environment variables in nginx configuration" heading).  I got as far as being able to at least set an environment variable to the output of that command (e.g., `DOCKER_MACHINE_IP=$(docker-machine ip default)`), but I couldn't get the config file to be valid.